### ISSUE TITLE
Global Shortcut responding to keys that I have not registered - Hotkey 'STRG+<'

### ISSUE THREAD

**Issue by borsTiHD** [wilix-team/iohook#197]
Hi all,

I have a problem with my registered global shortcut.
My app should react after I pressed the following hotkey 'STRG+<'.
I'm using 'qwertz' layout and the keys are next to each other on the keyboard.

My shortcut is working, thats not the problem.
The shortcut also reacts to another key combination which is often used by us with another program. This is the hotkey I don't want to react to: 'Win+<'

Edit:
I have a second question.
Is there a good way to prevent multiple casting my shortcut if the user is pressing the combination for 1-2 seconds? Or is it possible to do a check like `while ( stillPressed(x) ) { //wait }`?

## Expected Behavior
I only want my shortcut to react to the following key combination: 'STRG+<'

## Current Behavior
My shortcut reacts to other key kombinations as well.
For example: 'Win+<'

## Steps to Reproduce (for bugs)
```
const ioHook = require('iohook')
const globalHotkey = ioHook.registerShortcut([29, 0], (keys) => { //do something }
ioHook.start()
```

## Your Environment
* iohook: ^0.6.1
* Electron: ^6.0.11
* NPM: 6.10.0
* Nodejs: v12.7.0
* Operating System and version (desktop or mobile): Win10
* Link to your project: *private sorry*

Thanks for helping. :)

**Comment by borsTiHD** [comment]
I think I solved my own problem. :D 
I have noticed that the function started by the hotkey causes problems.

This function casts a childprocess with a autoit script to capture marked text anywhere.
(this is how it goes... you mark some text, use the hotkey mentioned above and the autoit script send 'STRG+C' to copy the marked text)

After a few trys with the hotkey, it seems that my ioHook hotkey reacts to every keypress with 'STRG' involved.

After I looked at the source code of iohook, I change the registration of the hotkey to the following:
```
const globalHotkey = ioHook.registerShortcut([29, 0], (keys) => { console.info('Keydown Event') }, (keys) => { //do something }
```
That way, it only reacts after the keyup event and I hope nothing disturbs this process. :)

But why is the "keyup" callback not in the documentation?
